src/js/views/hallOfFame.js

```diff
@@ -42,13 +43,19 @@ async function updatePlayers(inputs, updateEvents) {
                 }
             }
 
-            players[i].bestStats = {
-                gp: 0,
-                min: 0,
-                per: 0,
-            };
+            players[i].bestStats = {};
+            let bestEWA = 0;
             for (let j = 0; j < players[i].stats.length; j++) {
-                if (players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per > players[i].bestStats.gp * players[i].bestStats.min * players[i].bestStats.per) {
+                const rEWA = players[i].stats[j].ewa;
+                let pEWA = 0;
+                for (let k = 0; k < players[i].statsPlayoffs.length; k++) {
```

Comment:
@dumbmatter I think I found a slight bug in my own code.

What happens if a player is traded mid-season and makes the playoffs with the new team?

Will the `stats` array have two entries for the year (one for each team) and `playoffStats` array have one entry for the team they had playoffs with?

This inner for loop will double-count the second playoffs, I think.